src/components/form-attachment/list.vue

```diff
@@ -19,6 +19,22 @@ except according to the terms contained in the LICENSE file.
       <p>{{ $t('heading[0]') }}</p>
       <p>{{ $t('heading[1]') }}</p>
     </div>
+    <div v-if="datasetLinkable" class="panel-dialog">
+      <div class="panel-heading">
+        <span class="panel-title">
+          <span class="icon-database"></span>
+          {{ $t('datasetsPreview.title') }}
+        </span>
+      </div>
+      <div class="panel-body">
+        <i18n-t tag="p" keypath="datasetsPreview.body.full">
+          <template #documentation>
+            <!-- TODO. Specify the `to` prop. -->
+            <doc-link>{{ $t('datasetsPreview.body.documentation') }}</doc-link>
+          </template>
+        </i18n-t>
```

Comment:
I mentioned this during the meeting, but could you use `moreInfo.clickHere` or `moreInfo.helpArticle` for the last sentence? In that case, `datasetsPreview.body` won't use component interpolation (it won't include `{documentation}`) and will just be a simple message. You could then use `<sentence-separator>` between `$t('datasetsPreview.body')` and the `<i18n-t>` block for `moreInfo.clickHere`/`moreInfo.helpArticle`.